pkg/wwan/mm-init.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
```

Comment:
How about putting a Copyright and License line after line 1?

Reply:
Done